### PR TITLE
Embedded Systems QA Changes

### PR DESCRIPTION
**Pull request by bill-luu**
Will update more as full system testing continues.

### CODE REVIEW

**Comment by bill-luu** [comment]
Driver Controls - CCS Testing Completed

File: EpsilonCCS/Src/CanParsers/CanParser.c [hunk 1]
```diff
@@ -80,11 +87,14 @@ void HAL_CAN_RxCpltCallback(CAN_HandleTypeDef* hcan)
     msg->StdId = hcan->pRxMsg->StdId;
 
     memcpy(msg->Data, hcan->pRxMsg->Data, 8);
+    osMessagePut(canRxQueueId, (uint32_t)msg, 0);
 
-    if (osMessagePut(canRxQueueId, (uint32_t)msg, 0) == osOK)
+    if (HAL_CAN_Receive_IT(hcan, CAN_FIFO0) == osOK)
```
Comment (enochtsang) [inline EpsilonCCS/Src/CanParsers/CanParser.c]: Why isn't the queue being used? Doesn't this have race condition issues with just straight using the handler?

Comment (bill-luu) [reply EpsilonCCS/Src/CanParsers/CanParser.c]: Where's still being used (line 90), but I wanted the LEDs to tell us about the recieved function.

**Comment by bill-luu** [comment]
Lights confirmed working with Driver Controls and CCS!

**Pull request merged**